_spinnaker_install_admin_guides/secrets-vault.md

```diff
@@ -43,7 +43,9 @@ secrets:
 ```
 
 ## Configuring Halyard to use Vault secrets
-The Halyard daemon will need access to the Vault server in order to decrypt secrets for validation and deployment. So you'll also need to add the config block to your `halyard.yml`:
+Halyard will need access to the Vault server in order to decrypt secrets for validation and deployment. Most Spinnaker users are familiar with the `~/.hal/config` file, which contains all the settings for the Spinnaker services. However, there also exists the lesser-known `halyard.yml`, which configures the Halyard daemon itself. You can find this file in `/opt/spinnaker/config`.
```

Comment:
Funny style. I'd just say that you can configure Halyard in halyard.yml, where it's found, and that if you run Halyard Armory from its docker image, you'll need to mount the volume containing the new config.